scripts/deploy.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -euE
+
+BRANCH_NAME="${DEPLOY_BRANCH:=main}"
+REPO_URL="${DEPLOY_REPO:=https://github.com/fleetyards/ExFleetYards}"
+USER="${DEPLOY_USER:=fleetyards-api}"
+RELESES_TO_KEEP="${RELEASES_TO_KEEP:=5}"
+ENVIRONMENT="${DEPLOY_ENV:=staging}"
+
+HOME=/home/$USER
+RELESES_DIR=$HOME/releases
+DEPLOYMENT_TIME=$(date +"%Y%m%d%H%M%S")
+DEPLOY_TO=$RELESES_DIR/$DEPLOYMENT_TIME
+CONFIG_PATH=$HOME/config.exs
+
+function prepare() {
+  echo "-> Prepare Deployment"
+  if [ -f "$HOME/CURRENT_RELEASE" ]; then
+    mv $HOME/CURRENT_RELEASE $HOME/PREVIOUS_RELEASE
+  fi
+
+  echo $DEPLOYMENT_TIME >> $HOME/CURRENT_RELEASE
+}
+
+function run() {
+  echo "-> Start Deployment"
+  echo "--> Checkout Source into $DEPLOY_TO"
+
+  git clone --branch $BRANCH_NAME $REPO_URL $DEPLOY_TO
+
+  pushd $DEPLOY_TO > /dev/null
+
+  echo "--> Get deps"
+
+  MIX_ENV=$ENVIRONMENT FLEETYARDS_CONFIG_PATH=$CONFIG_PATH mix deps.get --only prod
```

Comment:
The runtime services provider only works once a Release is started, so can be omitted here